acme_diags/parameter/enso_diags_parameter.py

```diff
@@ -0,0 +1,27 @@
+from .core_parameter import CoreParameter
+
+
+class EnsoDiagsParameter(CoreParameter):
+    def __init__(self):
+        super(EnsoDiagsParameter, self).__init__()
+        # A list of the reference names to run the diags on.
+        self.ref_names = []
+        self.ref_timeseries_input = True
+        self.test_timeseries_input = True
+        # Granulating with regions doesn't make sense,
+        # because we have multiple regions for each plot.
+        # So keep all of the default values except regions. 
+        #self.seasons = ['ANN']
+        #print(dir(self))
+        self.granulate.remove('seasons')
```

Comment:
I recall that there was documentation on this on the E3SM Diags documentation page, but I can't seem to find it. The actual code is a part of `cdp`: https://github.com/CDAT/cdp/blob/master/cdp/cdp_parser.py#L380

It was basically done to "break down" parameters with many long list attributes into multiple parameter objects with shorter lists. Since the multiprocessing is done in a round-robin fashion, it'll was seen that this improves the performance, where one process doesn't end up with expensive diags.

So by default, you granulate based on `variables`, `seasons`, `plevs`, `regions`. For some reason, you guys decided not to granulate based on the `seasons` parameter.